README.md

```diff
@@ -24,7 +24,7 @@ grunt.loadNpmTasks('grunt-contrib-qunit');
 ## Qunit task
 _Run this task with the `grunt qunit` command._
 
-You have chosen to write your unit tests using [QUnit][], you have written a
+You have chosen to write your unit tests using [QUnit](http://qunitjs.com/), you have written a
```

Comment:
Thank you for the instructions.  The first command fails:

```
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.
```
Trying to figure it out now.